sdapsbase.dtx

```diff
@@ -84,6 +84,20 @@
 \ExplSyntaxOn
 %    \end{macrocode}
 %
+% We first adress the fact that some primitives do not have the same name under Lua\LaTeX{} and earlier \LaTeX{}es.
+%    \begin{macrocode}
+\sys_if_engine_luatex:TF
+  {
+    \cs_set_eq:NN \sdapssavepos \savepos
```

Comment:
Maybe this should be `\sdaps_savepos` then to be closer to LaTeX 3 style names? Not sure about adding an `_` in front.